src/test/cypress/integration/exam/ExamCreationDeletion.spec.ts

```diff
@@ -44,20 +45,28 @@ describe('Exam creation/deletion', () => {
         creationPage.setEndText('Cypress exam end text');
         creationPage.setConfirmationStartText('Cypress exam confirmation start text');
         creationPage.setConfirmationEndText('Cypress exam confirmation end text');
-        creationPage.submit().its('response.statusCode').should('eq', 201);
+        creationPage
+            .submit()
+            .then((examResponse) => {
+                examId = examResponse.response!.body.id;
+            })
+            .its('response.statusCode')
+            .should('eq', 201);
```

Comment:
A small performance improvement. The status code is always available when the request response is available. If the status code is not 201 then cypress will wait for 20 seconds expecting the status code to change to 201. This of course will never happen. By moving the assertion into the `then` block this can be avoided.
```suggestion
        creationPage
            .submit()
            .then((examResponse: any) => {
                examId = examResponse.response.body.id;
                expect(examResponse.response.statusCode).to.eq(201)
            });
```